pkg/storage/stores/shipper/bloomshipper/shipper.go

```diff
@@ -15,6 +15,24 @@ import (
 	"github.com/grafana/loki/pkg/storage/stores/shipper/bloomshipper/config"
 )
 
+type Interval struct {
+	Start model.Time
```

Comment:
nit: Maybe we can reuse model.Interval instead?

Reply:
I started with using `model.Interval`, but I wanted to extend it with `Cmp()` and `String()`.